spec/index.bs

```diff
@@ -9738,23 +9754,26 @@ dictionary GPUCommandEncoderDescriptor
                         - The set of texture regions in |attachmentRegions| must be pairwise disjoint.
                             That is, no two texture regions may overlap.
                     </div>
-                1. Consider each [=texture subresource=] in |attachmentRegions| to be used as
-                    an [=internal usage/attachment=] for the duration of the render pass.
+                1. [$usage scope/Add=] each [=texture subresource=] in |attachmentRegions|
+                    to |pass|.{{GPURenderCommandsMixin/[[usage scope]]}}
+                    with usage [=internal usage/attachment=].
 
                     If a subresource is seen more than once, consider it used only once.
```

Comment:
This note could be reworded, to say that overlaps are checked above, but not require deduplication. Then we'd change [1] to remove the requirement that only a single attachment usage is present.

Reply:
I'll avoid changing this now but file an issue or maybe a PR for this.

Reply:
#4799